tox.ini

```diff
@@ -22,6 +22,11 @@ commands =
 [testenv:venv]
 commands = {posargs}
 
+[testenv:cover]
+commands =
+    coverage run setup.py test {posargs}
```

Comment:
So my only issue with this is it's running testr and not stestr. I think it would be ideal if we used stestr to run the testing for coverage too.